src/media/js/routes.js

```diff
@@ -34,6 +34,11 @@ define('routes',
         {'pattern': '^/usage$', 'view_name': 'usage'},
     ]);
 
+    var search = '/api/v2/fireplace/search/?cache=1&vary=0';
+    if (settings.meowEnabled) {
+        search = '/api/v2/multi-search/?cache=1&vary=0';
```

Comment:
I think it'd be easier for the frontend if it was able to pass a param. Otherwise we would have to switch between completely different endpoints based on the user's selection. It also matches what we do for device filtering and such.